packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added the ability to download job spool files using other codepages with `--encoding` on the `zowe jobs download output`, `zowe jobs view spool-file-by-id` and `zowe jobs view all-spool-content` commands.  [#1822](https://github.com/zowe/zowe-cli/pull/1822)
```

Comment:
Could we add a benefit to the user to this note? What does this change accomplish?